lib/core/public/a11y-check.js

```diff
@@ -16,6 +16,10 @@ axe.a11yCheck = function (context, options, callback) {
 		options = {};
 	}
 
+	if (!('iframes' in options)) {
```

Comment:
Actually you could get rid of this code simply by negating the test in `run-rules.js`